extension/scripts/features/travelfillmax/ttTravelFillMax.js

```diff
@@ -0,0 +1,54 @@
+"use strict";
+
+(async () => {
```

Comment:
Should add a check that you are abroad (shouldn't run while flying).

Reply:
You are still registering the feature, instead of preventing it from even registering.